useful_scripts/rg_replace.sh

```diff
@@ -24,6 +24,14 @@
 #   3. `gs_rgr`
 #   4. `gs_rg_replace`
 
+# SIMPLE FISH IMPLEMENTATION (not using bins)
+# essentially we create a function that executes rg_replace.sh (named rgr.sh in my config) 
+
+# function rgr
+#   set -l DOOTFILE_LOC ~/Documents/dotFiles/ # replace to the current location of rg_replace.sh
```

Comment:
`DOOTFILE_LOC` should be `DOTFILE_LOC`, no? I recommend changing "replace" to "set", as well.

Reply:
My mistake, I often refer to my dotfiles as my doots for added confusion.